bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/GstarToTpTop/ZpTpt_Zp1500Nar_Tp1000NarLH_Ht/ZpTpt_Zp1500Nar_Tp1000NarLH_Ht_proc_card.dat

```diff
@@ -0,0 +1,15 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
+define p = g u c d s u~ c~ d~ s~
```

Comment:
is worth removing the proton and jet definitions to avoid mismatches with the model...